docs/proposals/rename-group.md

```diff
@@ -0,0 +1,162 @@
+
+### Enable the use of topolvm.cybozu.com
+
+We will make the following changes to allow users to continue to use `topolvm.cybozu.com`.
+
+#### Addition of group name setting function
+
+- If you set the `USE_LEGACY` environment variable, the group name `topolvm.cybozu.com` will be used
+- If the `USE_LEGACY` environment variable is not set, the group name `topolvm.io` will be used
+
+#### Group name setting in helm chart
+
+Add the `.Values.useLegacy` variable to the helm chart and set this variable to `true` to set the installation manifest to use `topolvm.cybozu.com`.
+If the `.Values.useLegacy` variable is not set to `true`, the group name will be `topolvm.io`.
+
+If you are going to set the following variables, you may need to set them appropriately to use `topolvm.io` or `topolvm.cybozu.com` in the variables.
+
+- `.Values.storageClasses`
+- `.Values.node.volumes`
+- `.Values.volumeMounts.topolvmNode`
+
+#### Automatic generation for API and CRD
+
+Automatically generating the API and CRD of `topolvm.cybozu.com` based on the API and CRD of `topolvm.io`.
+
+### Auto group conversion for LogicalVolume
+
+Add a wrap client for controller-runtime client to automatically convert a group of LogicalVolume resources.
+Change each controller and other components to use this wrapped client.
+
+### Things to be done by users of topolvm.cybozu.com for upgrading the helm chart
+
+Since this change was released, users of TopoLVM with the group name `topolvm.cybozu.com` will need to take the following actions when upgrading the helm chart.
+
+- Set the `.Values.useLegacy `variable to` true` in `values.yaml` in the helm chart
+- If there is a part of the `values.yaml` in the helm chart that uses the string `topolvm.cybozu.com`, change it to `topolvm.io`
+
+If you upgrade the helm chart without taking above actions, it is possible that TopoLVM related resources will be deleted.
+
+### Migrate from topolvm.cybozu.com to topolvm.io
+
+If you are already using TopoLVM with the group name topolvm.cybozu.com and want to migrate the group name to topolvm.io after the release of this change, you can manually migrate the data by the following method.
+
+**You should back up LVM data managed by TopoLVM because the data could be lost when the migration is failed.**
+
+**Please test the migration procedure before you execute the migration. Also, since the migration procedure is complicated, please consider using TopoLVM as topolvm.cybozu.com without migration.**
+
+1. Deploy the `topolvm.io` version of TopoLVM (when used with the `topolvm.cybozu.com` version of TopoLVM,  names of cluster-wide resources such as StorageClass names need to be changed)
```

Comment:
Would you please add a note to suggest deploying a new topolvm to a different namespace from the existing topolvm to avoid collisions of resource names?